changelog/next-release.md

```diff
@@ -156,3 +156,16 @@ type RemovedInput = {
 ### 14. Removed method `sdk.vault.getScorePercentiles`
 
 ---
+
+### 15. Added method
+### `sdk.osToken.getLeverageStrategyProxy`
+
```

Comment:
i think you can add some description to new methods from 
`src/methods/osToken/index.ts`